lib/rspec/core/option_parser.rb

```diff
@@ -140,8 +140,18 @@ def parser(options)
           options[:full_backtrace] = true
         end
 
-        parser.on('-c', '--[no-]color', '--[no-]colour', 'Enable color in the output.') do |o|
-          options[:color] = o
+        parser.on('-c', '--color', '--colour', 'Deprecated: enable color in the output.') do |_o|
+          options[:color] = true
+        end
+
+        parser.on('--force-color', '--force-colour', 'Force the output to be in color.') do |_o|
+          options[:force_color] = true
+          options.delete(:no_color)
```

Comment:
It sounds like this would make the first flag win instead of last flag wins -- because when the first of the two flags is processed, it deletes the other one, and the last flag is therefore never processed.

One alternate idea is to abort with a message telling the user to set only one of `--no-color` and `--force-color`.  I think I like that better -- instead of guessing their intent, make them correct the command.  That could be done by adding a validation to the `parse` method (or to a new `validate` helper method called from `parse`) that `abort`s with a useful message if both `option[:no_color]` and `options[:force_color]` are `true`.